README.rst

```diff
@@ -654,6 +657,40 @@ Example::
 Response::
     Binary file
 
+@@send-booking-reminders
+------------------------
+
+This view sends a booking reminder email to all the bookings inside of PrenotazioniFolders which have the Reminder Notification Gap field popolated.
+If you mind to setup a cronjob to call this view, you migth use a special script call. The script is located at src/redturtle/prenotazioni/scripts/notify_upcoming_bookings.py
```

Comment:
typo

Reply:
please add a buildout example on how to configure a cron with this script and how to send logs into a separate log file to not fill the standard one with unuseful infos

Reply:
Done